api/krusty/sortordertransformer_test.go

```diff
@@ -327,5 +327,5 @@ sortOptions:
 
 	kustOptions := th.MakeDefaultOptions()
 	err := th.RunWithErr("base", kustOptions)
-	require.ErrorContains(t, err, "unable to load builtin SortOrderTransformer.builtin.[noGrp]")
+	require.ErrorContains(t, err, "unable to load builtin SortOrderTransformer.[noVer].builtin")
```

Comment:
Just leaving myself a note here. It looks like we're changing `builtin` from being the "verison" to being the "group". I think that's correct, so thank you for that change.